icepyx/tests/test_behind_NSIDC_API_login.py

```diff
@@ -23,6 +24,7 @@ def reg():
 
 @pytest.fixture(scope="module")
 def session(reg):
+    os.environ = {"EARTHDATA_USERNAME": "icepyx_devteam"}
```

Comment:
This is probably beyond the PR but have you all considered having a "default" username in case users want to use icepyx but don't have an EDL login?  (don't quote me =) )

Reply:
We have, and we were told in no uncertain terms not to do it (it would also be very hard to do without somehow exposing the password). Since it's an ubiquitous problem, as you are well aware, we opted to just have users get an EDL (since they would have to just about anywhere except OA and SlideRule, I think), and we store the secrets for a dev account for CI testing (which isn't accessible for PRs from forks either).